app/components/course-page/completed-step-notice.hbs

```diff
@@ -40,4 +40,24 @@
       </PrimaryLinkButton>
     {{/if}}
   </div>
+
+  {{#if (gt this.nextStep.globalPosition 4)}}
```

Comment:
this is the best way i could find to make this work. can you confirm that global position 4 is consistently the first stage of each challenge?

Reply:
This might be true currently, but it can easily change in the future (maybe we add another "onboarding" step or so) - it'd be very difficult for us to remember to come and update this usage. 

I'd do something like: 

```
{{#if (and (eq this.currentStep.type "CourseStageStep") (not this.currentStep.courseStage.isFirst))}}
```